### DISCUSSION TITLE
Misisng models after initial installation.

### DISCUSSION THREAD

**Discussion by headkit** [Stability-AI/StableStudio#105] [Q&A]
I installed the app and run it locally with  'stable-diffusion-webui' on localhost:3000. Now I wanted to select a model but the list is empty.

In the settings section there is a HOST URL form field telling me that http://127.0.0.1:7861/ is the usual URL. Why then does the server start with port 3000? And where do I find the models or where can I set a directory to the model weights?

Thank you!

**Comment by barepixels** [comment]
I store all of my models on a faster drive (SSD)  Here is how to instruct Confyui to use my external model folder

#Rename this to extra_model_paths.yaml and ComfyUI will load it

#config for a1111 ui
#all you have to do is change the base_path to where yours is installed
a111:
    base_path: D:\stable-diffusion-webui

    checkpoints: D:\stable-diffusion-shared\Ckpt
    configs: models/Stable-diffusion
    vae: D:\stable-diffusion-shared\VAE
    loras: |
         D:\stable-diffusion-shared\Lora
         models/LyCORIS
    upscale_models: |
                  models/ESRGAN
                  models/SwinIR
    embeddings: D:\stable-diffusion-shared\Embeddings
    hypernetworks: models/hypernetworks
    controlnet: models/ControlNet

#other_ui:
#    base_path: path/to/ui
#    checkpoints: models/checkpoints
#    gligen: models/gligen
#    custom_nodes: path/custom_nodes

**Reply by barepixels** [reply]
hmm github didnt paste right.  anyway do research on  extra_model_paths.yaml

**Comment by KAJdev** [comment]
Are you sure you installed the correct version? The `main` branch does not have a comfyUI integration. You might be looking for the [`tauri`](https://github.com/Stability-AI/StableStudio/tree/tauri) branch

**Comment by headkit** [comment]
Whats the difference of these branches?
